core/examples/luigi-sample-angular/src/app/overview/overview.component.html

```diff
@@ -1,36 +1,40 @@
 <header class="fd-page__header">
-  <div class="fd-action-bar">
-    <div class="fd-action-bar__header">
-      <h1 class="fd-action-bar__title">
-          Luigi Angular Example
-      </h1>
-      <p class="fd-action-bar__description">Follow the links below to the Luigi feature demos.</p>
-    </div>
-  </div>
+	<div class="fd-action-bar">
+		<div class="fd-action-bar__header">
+			<h1 class="fd-action-bar__title">
+				Luigi Angular Example
+			</h1>
+			<p class="fd-action-bar__description">Follow the links below to the Luigi feature demos.</p>
+		</div>
+	</div>
 </header>
 
 <section class="fd-section">
-  <div class="fd-section__header">
-    <h1 class="fd-section__title">LuigiClient Features</h1>
-  </div>
-  <div class="fd-panel">
-      <ul class="fd-list-group">
-        <li class="fd-list-group__item clickable" *ngFor="let item of luigiClientLinks" (click)="luigiClient.linkManager().navigate(item.link)">
-          <strong>{{item.text}}</strong> &nbsp;&ndash; {{item.description}}
-        </li>
-      </ul>
-  </div>
+	<div class="fd-section__header">
+		<h1 class="fd-section__title">LuigiClient Features</h1>
+	</div>
+	<div class="fd-panel">
+		<ul class="fd-list-group">
+			<li class="fd-list-group__item clickable" *ngFor="let item of luigiClientLinks" (click)="luigiClient.linkManager().navigate(item.link)">
+				<strong>{{item.text}}</strong> &nbsp;&ndash; {{item.description}}
+			</li>
+		</ul>
+	</div>
 </section>
 
 <section class="fd-section">
-  <div class="fd-section__header">
-    <h1 class="fd-section__title">Luigi Core Features</h1>
-  </div>
-  <div class="fd-panel">
-      <ul class="fd-list-group">
-        <li class="fd-list-group__item clickable" *ngFor="let item of luigiCoreLinks" (click)="luigiClient.linkManager().navigate(item.link)">
-          <strong>{{item.text}}</strong> &nbsp;&ndash; {{item.description}}
-        </li>
-      </ul>
-  </div>
-</section>
+	<div class="fd-section__header">
+		<h1 class="fd-section__title">Luigi Core Features</h1>
+	</div>
+	<div class="fd-panel">
+		<ul class="fd-list-group">
+			<li class="fd-list-group__item clickable" *ngFor="let item of luigiCoreLinks" (click)="luigiClient.linkManager().navigate(item.link)">
+				<strong>{{item.text}}</strong> &nbsp;&ndash; {{item.description}}
+			</li>
+			<li class="fd-list-group__item clickable" (click)="sendDirtyEvent()">
+				<strong>Pretend there are unsaved changes</strong> &nbsp;&ndash; shows a confirmation modal when user wants to leave
```

Comment:
Please provide better guiding text that makes more straightforward to understand how to test the feature and actually test it.